release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -101,16 +101,47 @@ With this release, the installation program uses Cluster API instead of Terrafor
 [id="ocp-4-17-web-console_{context}"]
 === Web console
 
+[id="ocp-4-17-ocp-lightspeed_{context}"]
+==== {ols} is available in the web console
+
+Starting with {product-title} 4.16, {ols} is available for use in the web console.
+
+//https://github.com/openshift/openshift-docs/pull/81581
+//For more information, see xref:../web_console/capabilities_products-web-console.html#openshift-lightspeed-web-console_capabilities-web-console[Enhancing your experience in the {product-title} web console by using {ols-official}].
+
 [id="ocp-4-17-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
 
+* Deprecated Operators are displayed in the OperatorHub before and after installation.
+* Config files for `MachineConfig` objects were moved to a new tab enabling you to check the content of the configuation files without having to manually retrieve its contents.
+* An alert was added to `operator-hub-item-details` and `operator-hub-subscribe` components if your cluster is on {gcp-first} with Workload Identity Foundation (WIF).
```

Comment:
```suggestion
* An alert will be shown on Operator details and Operator installation page if your cluster is on {gcp-first} with Workload Identity Foundation (WIF).
```

Reply:
**operator details modal**
![Screenshot 2024-09-25 at 10 48 04 AM](https://github.com/user-attachments/assets/1ca46679-b761-4c3b-8551-647361787214)

**operator installation page**
![Screenshot 2024-09-25 at 10 48 09 AM](https://github.com/user-attachments/assets/593b211b-a0e9-4beb-b2b3-85fea83b8a8e)

Reply:
I see what you are saying! Okay, that makes sense! Thanks!